README.md

```diff
@@ -1,6 +1,5 @@
 # rebar3_format
 
-[![Build Status](https://api.travis-ci.com/AdRoll/rebar3_format.svg?branch=main)](https://travis-ci.com/AdRoll/rebar3_format)
```

Comment:
You can still keep this, but not in this form, though.

Check this out: 

```
[![Build Status](https://github.com/AdRoll/rebar3_format/workflows/build/badge.svg)](https://github.com/AddRoll/rebar3_format)
```

Reply:
Great TY